### DISCUSSION TITLE
The toolbar should be hidden early

### DISCUSSION THREAD

**Discussion by nana123126** [ppy/osu#25114] [Ideas] [answered]
When resuming the game after pausing, it should be hidden (i.e. before clicking on the orange cursor)

The reason is that the toolbar will change its zoom when hidden, and it will synchronize with the start of the game, which will disrupt the player's order. Continuing the game does not change the cursor position, so the cursor moves relatively downward after zooming, which may affect balance under some special zooming conditions.

Another method is to not automatically hide the toolbar or not allow players to open it during the game.

It now operates like this:
https://github.com/ppy/osu/assets/87159802/0ead882e-d9ce-413c-9688-c31166258d43

Special zoom:              
https://github.com/ppy/osu/assets/87159802/53e1bc9a-decd-4de0-b004-9a893715a0d0

**Comment by peppy** [comment]
Pausing doesn't bring the toolbar up. Are you manually showing it?

**Reply by nana123126** [reply]
Yes, I use 'Ctrl T' to display it and perform other operations, but sometimes I forget to close it.

**Comment by peppy** [comment, marked as answer]
I think getting this right would involve bringing in the updated toolbar design, and potentially *not* having the toolbar resize the whole game (but instead appear as an overlay, similar to settings to notification). I've been considering this for a while, but been waiting for the updated design to take action.

I also want the toolbar to be accessible via putting the cursor near the top of the screen, or potentially swiping on mobile.
